### DISCUSSION TITLE
allow_llm_to_see_data is set to False in VannaBase.generate_sql. why?

### DISCUSSION THREAD

**Discussion by mukesh1811** [vanna-ai/vanna#518] [Q&A]
allow_llm_to_see_data is set to False in VannaBase.generate_sql. why?

**Comment by zainhoda** [comment]
Please see the data security FAQ:

https://vanna.ai/data-security-faq.html

> Database contents are not sent to Vanna's servers or the LLM unless you specifically set the parameter allow_llm_to_see_data = True in the built-in Flask app or use functions like vn.generate_summary explicitly that require the LLM to "see" the data in order to produce an answer. This parameter is set to False by default.
